sys/include/div.h

```diff
@@ -128,6 +196,73 @@ static inline uint64_t div_u64_by_15625div512(uint64_t val)
 }
 
 /**
+ * @brief Divide val by (625/576)
+ *
+ * This is used to quantize a 1MHz value to the closest 921600Hz value,
+ * e.g., for timers.
+ *
+ * @param[in]   val     dividend
+ * @return      (val / (625/576))
+ */
+static inline uint64_t div_u64_by_625div576(uint64_t val)
+{
+    /*
+     * This saves around 1400 bytes of ROM on Cortex-M platforms (both ARMv6 and
```

Comment:
Oops, copy-paste, I'll check if it does but I think it makes exactly the same as before. Moreover, it makes sense to take care of the overflow, I can just remove the comment but the code seems ok for me.